beacon_node/store/src/memsize.rs

```diff
@@ -0,0 +1,44 @@
+use crate::metrics::BEACON_STATE_MEMORY_SIZE_CALCULATION_TIME;
+use milhouse::mem::{MemorySize, MemoryTracker};
+use std::time::Instant;
+use types::{BeaconState, EthSpec};
+
+/// BeaconState Wrapper for memory tracking.
+pub struct BeaconStateWrapper<'a, E: EthSpec>(pub &'a BeaconState<E>);
+
+impl<'a, E: EthSpec> MemorySize for BeaconStateWrapper<'a, E> {
+    fn self_pointer(&self) -> usize {
+        self.0 as *const _ as usize
+    }
+
+    fn subtrees(&self) -> Vec<&dyn MemorySize> {
+        vec![]
```

Comment:
You're missing all of the fields of the `BeaconState`, i.e. the interesting parts. See my previous draft PR:

https://github.com/michaelsproul/lighthouse/blob/e34fafab5977faae5362c3cc16f969256769502e/consensus/types/src/beacon_state/memsize.rs#L17-L90